testing/regress/ecl/dynamicoptflag.ecl

```diff
@@ -34,7 +34,7 @@ count(p);
 output(p(f='NOT'));
 count(p(f='NOT'));
 
-p2 := PRELOAD(dataset(DYNAMIC(prefix + 'no:_such_file_again'), {string10 f}, FLAT, OPT), 2);
+p2 := PRELOAD(dataset(DYNAMIC(prefix + 'no_such_file_again'), {string10 f}, FLAT, OPT), 2);
```

Comment:
Should we add in a new regression test for this ?

Reply:
will extend unittests to test invlid paths like this